tests/network/hotplug.go

```diff
@@ -547,3 +555,13 @@ func removeInterface(vm *v1.VirtualMachine, name string) error {
 	_, err = kubevirt.Client().VirtualMachine(vm.Namespace).Patch(context.Background(), vm.Name, types.JSONPatchType, patchData, &metav1.PatchOptions{})
 	return err
 }
+
+func lookupNetworkByName(networks []v1.Network, name string) *v1.Network {
```

Comment:
This fits under `libnet`.
Perhaps you can move there a function that returns the interface and network, with a boolean that can be examined. It will just get as input the VM/VMI name, namespace and logical network name.

Reply:
Moved to libnet.